source/_integrations/airios.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: Airios
+description: Instructions on how to integrate Airios ventilation units into Home Assistant.
+ha_category:
+  - Climate
+  - Fan
+ha_config_flow: true
+ha_release: 2025.4
+ha_iot_class: Local Polling
+ha_domain: airios
+ha_platforms:
+  - fan
+ha_codeowners:
+  - '@scabrero'
+ha_integration_type: hub
+---
+
+The **Airios** {% term integration %} allows you to control and monitor ventilation units and accessories from different manufacturers.
+
+**Airios** develops and produces components for residential ventilation systems that final manufacturers use to build their products upon, from controller boards to remote controls or sensors. These components communicate over a proprietary RF protocol from Honeywell called Ramses II in the 868Mhz band.
+
+
+This {% term integration %} has been tested with the following manufacturers.
```

Comment:
will you provide a list here? Otherwise I would remove this statement.

Reply:
Hi, thanks for review. Yes, the idea is to provide a list where each entry is a link to its virtual integration page. And in each virtual integration page list the compatible/tested models and accessories.

Reply:
aha, ok. thank you for clarifying.